docs/getting-started/gs-faqs.md

```diff
@@ -173,7 +184,7 @@ The system should follow the [email normalization rules](gs-normalization-encodi
 Yes. Not storing mappings may increase processing time drastically when you have to map millions of email addresses or phone numbers. Recalculating only those mappings that actually need to be updated, however, reduces the total processing time because only about 1/365th of UID2s need to be updated daily.
 
 :::info
-Unless you are using a private operator, you must map email addresses, phone numbers, or hashes consecutively, using a single HTTP connection, in batches of 5,000 emails at a time. In other words, do your mapping without creating multiple parallel connections. 
+Unless you are using a <Link href="../ref-info/glossary-uid#gl-private-operator">Private Operator</Link>, you must map email addresses, phone numbers, or hashes consecutively, using a single HTTP connection, in batches of 5,000 emails at a time. In other words, do your mapping without creating multiple parallel connections. 
```

Comment:
Why don't we add the last line into the mod on docs/endpoints/post-identity-map.md ? It adds to the clarity i think.

 _do your mapping without creating multiple parallel connections._

Reply:
Good idea. Done, thx.